bin/git-delete-squashed-branches

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+targetBranch=$1
+
+if [[ -z $targetBranch ]]; then
+  targetBranch=$(git rev-parse --abbrev-ref HEAD)
+else
+  git checkout -q $targetBranch
```

Comment:
Better to mention the target branch will be checkout as a side effect.

Reply:
Should this be done by removing the `-q` flag or an explicit `echo`?

Reply:
removing the -q flag is enough.

Reply:
Done!